generate_rss.py

```diff
@@ -31,6 +36,63 @@ def findall(self, *args, **kwargs):
 entry.formatRFC2822 = feed.formatRFC2822 = _format_rfc_2822
 line_cache: dict[Path, dict[str, str]] = {}
 
+# Monkeypatch PEP and RFC reference roles to match Sphinx behaviour
+EXPLICIT_TITLE_RE = re.compile(r'^(.+?)\s*(?<!\x00)<(.*?)>$', re.DOTALL)
```

Comment:
As a script-level CONSTANT, shouldn't this be at the top? If not, then per PEP 8

```suggestion
EXPLICIT_TITLE_RE = re.compile(r'^(.+?)\s*(?<!\x00)<(.*?)>$', re.DOTALL)


```

Reply:
> As a script-level CONSTANT, shouldn't this be at the top? 

I'm working to add support for this upstream (in Docutils) -- this is intended as a temporary monkeypatch. All the code was copied from the same backport I used in `pep2html`. So it is at the top of the block, allowing easier deletion all at once when we can.

A

Reply:
Okay, that makes sense.